en/02_Developer_Guides/06_Testing/04_Fixtures.md

```diff
@@ -193,25 +197,7 @@ $team->Players()->add($john);
 > As the YAML fixtures will call `write`, any `onBeforeWrite()` or default value logic will be executed as part of the
 > test.
 
-## Fixtures for namespaced classes
-
-You will need to use fully qualified class names in your YAML fixture files. In the above examples, they belong to the global namespace so there is nothing requires, but if you have a deeper DataObject, or it has a relationship to models that are part of the framework for example, you will need to include their namespaces:
-
-```yml
-App\Test\Player:
-  john:
-    Name: join
-App\Test\Team:
-  crusaders:
-    Name: Crusaders
-    Origin: Canterbury
-    Players: =>App\Test\Player.john
-```
-
-> [!WARNING]
-> If your tests are failing and your database has table names that follow the fully qualified class names, you've probably forgotten to implement `private static $table_name = 'Player';` on your namespaced class. See [DataObject](api:SilverStripe\ORM\DataObject) for an example.
-
-## Defining many_many_extraFields
+## Defining `many_many_extraFields`
```

Comment:
This is unrelated refactoring - but I noticed that this is actually outdated. All the above examples use namespaced classes.
I've moved the warning up above.